### PR TITLE
Upgrade to 1.6.1

### PR DESCRIPTION
**Pull request by hcho3**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

### CODE REVIEW

**Comment by hcho3** [comment]
@conda-forge-admin, please rerender

**Comment by hcho3** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
Looks like the ARM build is failing with the R package. Maybe it is worth `cat`ing `config.log` to get more info?

```
** using staged installation
checking for aarch64-conda-linux-gnu-gcc... /home/conda/feedstock_root/build_artifacts/xgboost-split_1651298771780/_build_env/bin/aarch64-conda-linux-gnu-cc
checking whether the C compiler works... no
configure: error: in `/home/conda/feedstock_root/build_artifacts/xgboost-split_1651298771780/work/R-package':
configure: error: C compiler cannot create executables
See `config.log' for more details
ERROR: configuration failed for package ‘xgboost’
* removing ‘/home/conda/feedstock_root/build_artifacts/xgboost-split_1651298771780/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/R/library/xgboost’
```

**Comment by jakirkham** [comment]
Thanks Uwe! 🙏

Filed issue ( https://github.com/conda-forge/ctng-compiler-activation-feedstock/issues/74 ) with those details and cc'd both of you there. Please feel free to add/correct anything in the issue.

**Comment by apcamargo** [comment]
Anything else blocking this PR?

**Comment by hcho3** [comment]
@conda-forge-admin, please rerender

**Comment by hcho3** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
So the `r-base` that was fixed was `4.2.1`, but the `r-base` installed here is `4.0.5`. Are we able to upgrade the `r-base` used? Or do we need that fix backported to older `r-base` versions?

**Comment by hcho3** [comment]
@jakirkham It could be that `r-base` wasn't yet available on the CDN? Can you restart the test?

**Comment by jakirkham** [comment]
Don't think that is the issue. Think we need to backport the `r-base` fix to older `r-base` versions. Have already submitted PRs to do that ( https://github.com/conda-forge/r-base-feedstock/pull/213#issuecomment-1208775446 )

**Comment by hcho3** [comment]
Why is XGBoost feedstock picking up older r-base? I don't see any pins in the recipe.

**Comment by jakirkham** [comment]
Yeah I missed that as well initially. The comment linked above has references in it pertaining to the `r-base` versions used. While `r-base` has upgraded to `4.2`, the rest of the conda-forge stack has not yet made the move.

**Comment by hcho3** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,9 +17,10 @@ source:
     - 0001-Force-endian-flag-in-cross-compilation-mode.patch  # [arm64 or aarch64 or ppc64le]
 
 build:
-  number: 2
+  number: 0
   skip: true  # [win and cuda_compiler_version != "None"]
   skip: true  # [cuda_compiler_version == "10.2"]
+  skip: true  # [python_impl == 'pypy' and py < 38]
```
Comment (jakirkham) [inline recipe/meta.yaml:23]: Skipped due to the problem identified in this comment ( https://github.com/conda-forge/pypy3.6-feedstock/issues/83#issuecomment-1182510938 ), which also showed up in CI here

**Review by jakirkham — APPROVED** [review]
Thanks Philip! 🙏

**Pull request merged**